core/shared/src/main/scala/zio/FiberFailure.scala

```diff
@@ -27,10 +27,20 @@ import zio.stacktracer.TracingImplicits.disableAutoTrace
  * better integrate with Scala exception handling.
  */
 final case class FiberFailure(cause: Cause[Any]) extends Throwable(null, null, true, false) {
+
+  private var javaStackTrace: Array[StackTraceElement] = Thread.currentThread().getStackTrace
```

Comment:
Isn't it simpler to just change L29 so that the Throwable contains the stack trace?

```scala
final case class FiberFailure(cause: Cause[Any]) extends Throwable(null, null, true, true)
```